[2023-09-02T08:46:00.755+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=dag_with_postgres_operator_v02/run_id=manual__2023-09-02T08:45:59.622351+00:00/task_id=create_postgres_table permission to 509
[2023-09-02T08:46:00.756+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=dag_with_postgres_operator_v02/run_id=manual__2023-09-02T08:45:59.622351+00:00/task_id=create_postgres_table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dag_with_postgres_operator_v02/run_id=manual__2023-09-02T08:45:59.622351+00:00/task_id=create_postgres_table'
[2023-09-02T08:46:00.786+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=dag_with_postgres_operator_v02/run_id=manual__2023-09-02T08:45:59.622351+00:00/task_id=create_postgres_table permission to 509
[2023-09-02T08:46:00.787+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=dag_with_postgres_operator_v02/run_id=manual__2023-09-02T08:45:59.622351+00:00/task_id=create_postgres_table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dag_with_postgres_operator_v02/run_id=manual__2023-09-02T08:45:59.622351+00:00/task_id=create_postgres_table'
[2023-09-02T08:46:00.824+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v02.create_postgres_table manual__2023-09-02T08:45:59.622351+00:00 [queued]>
[2023-09-02T08:46:00.837+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v02.create_postgres_table manual__2023-09-02T08:45:59.622351+00:00 [queued]>
[2023-09-02T08:46:00.838+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-02T08:46:00.854+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2023-09-02 08:45:59.622351+00:00
[2023-09-02T08:46:00.861+0000] {standard_task_runner.py:57} INFO - Started process 590 to run task
[2023-09-02T08:46:00.864+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v02', 'create_postgres_table', 'manual__2023-09-02T08:45:59.622351+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/dag-postgres-operator.py', '--cfg-path', '/tmp/tmpain18sr0']
[2023-09-02T08:46:00.867+0000] {standard_task_runner.py:85} INFO - Job 61: Subtask create_postgres_table
[2023-09-02T08:46:00.927+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=dag_with_postgres_operator_v02/run_id=manual__2023-09-02T08:45:59.622351+00:00/task_id=create_postgres_table permission to 509
[2023-09-02T08:46:00.928+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=dag_with_postgres_operator_v02/run_id=manual__2023-09-02T08:45:59.622351+00:00/task_id=create_postgres_table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dag_with_postgres_operator_v02/run_id=manual__2023-09-02T08:45:59.622351+00:00/task_id=create_postgres_table'
[2023-09-02T08:46:00.931+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v02.create_postgres_table manual__2023-09-02T08:45:59.622351+00:00 [running]> on host 6c82fec646db
[2023-09-02T08:46:01.028+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='qpsss' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v02' AIRFLOW_CTX_TASK_ID='create_postgres_table' AIRFLOW_CTX_EXECUTION_DATE='2023-09-02T08:45:59.622351+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-02T08:45:59.622351+00:00'
[2023-09-02T08:46:01.030+0000] {sql.py:274} INFO - Executing: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        
[2023-09-02T08:46:01.041+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-02T08:46:01.052+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-02T08:46:01.057+0000] {sql.py:418} INFO - Running statement: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        , parameters: None
[2023-09-02T08:46:01.070+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v02, task_id=create_postgres_table, execution_date=20230902T084559, start_date=20230902T084600, end_date=20230902T084601
[2023-09-02T08:46:01.118+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-02T08:46:01.139+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
